mappings/net/minecraft/server/world/ServerChunkManager.mapping

```diff
@@ -27,6 +27,11 @@ CLASS uq net/minecraft/server/world/ServerChunkManager
 	METHOD a getWatchDistance (Lbcd;Lut;Z)I
 		ARG 0 chunkPos
 		ARG 1 player
+	METHOD a (Lbda;III)V
+		ARG 1 arg1
```

Comment:
any reason they are named as `arg1`? Is this a meaningful name?

Reply:
Enigma bug, should be resolved now though. Also should remove itself after load & save.